app/src/test/java/com/futurice/freesound/test/rx/IgnoringObserver.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.test.rx;
+
+import android.support.annotation.NonNull;
+
+import rx.Observer;
+
+/**
+ * An {@link Observer} which swallows all events without side effects.
+ */
+public class IgnoringObserver<T> implements Observer<T> {
```

Comment:
It wasn't my intention to make this class open, and I could go either way on the usefulness of being able to extend it. But I'll take the "closed by default" route and make it `final` as you've suggested.